services/status-api/app/public_content.py

```
@@ -0,0 +1,332 @@
+
+
+def _default_fetch(url: str, timeout: float) -> bytes:
+    request = urllib.request.Request(
+        url,
+        method="GET",
+        headers={
+            "Accept": "application/json",
+            "User-Agent": "ivrm-status-api/public-content-v1",
+        },
+    )
+    with urllib.request.urlopen(request, timeout=timeout) as response:
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)